packages/itwinui-react/src/core/RadioTiles/RadioTile.test.tsx

```diff
@@ -45,16 +45,87 @@ it('should render radio tile with icon', () => {
   expect(container.querySelector('.iui-radio-tile-icon')).toBeTruthy();
 });
 
-it('should take class and style', () => {
+it('should take tile class and style', () => {
```

Comment:
I think there could be one test that covers all the possible custom props that can be added instead of a separate test for each element. Thoughts @mayank99 ?

Reply:
agreed

Reply:
completed the refactor

Reply:
why not mount a single component that has all of those props? no need for creating a complex loop

Reply:
done